praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/service/PraktikumsstellenService.java

```diff
@@ -149,6 +158,105 @@ public void deleteAusbildungsPraktikumsstelle(UUID praktikumsstellenId) {
         }
     }
 
+    public void updateAusbildungsPraktikumsstelle(UUID praktikumsstellenId,
+            UpdateAusbildungsPraktikumsstelleWithMeldezeitraumAndAssignedNWKDto praktikumsstelleDto) {
+
+        AusbildungsPraktikumsstelle ausbildungsPraktikumsstelle = praktikumsstellenMapper.toEntity(praktikumsstellenId, praktikumsstelleDto);
+        ausbildungsPraktikumsstelle.setDienststelle(normalizeDienststelle(ausbildungsPraktikumsstelle.getDienststelle()));
```

Comment:
the statement looks like something that should be part of in serve method for saving